packages/composite-checkout/src/components/billing-fields.js

```diff
@@ -498,6 +437,8 @@ const DomainContactFieldsDescription = styled.p`
 
 function BillingFormSummary( { paymentData } ) {
 	const localize = useLocalize();
+	const billingData = paymentData.billing;
```

Comment:
This is fine but these can be shortened to `const { billingData, domainData } = paymentData;` One advantage is that using the destructuring form will throw an error if one of those keys doesn't exist, while the two line version will just happily assign `undefined` to a missing variable.